package.json

```diff
@@ -24,21 +24,23 @@
     "request": "^2.69.0"
   },
   "devDependencies": {
-    "hexo-cli": "^1.0.1"
+    "hexo-cli": "^1.0.1",
+    "multidep": "^2.0.0"
   },
   "hexo": {
     "version": "3.2.0"
   },
   "scripts": {
-    "start": "npm run dev",
+    "checklinks": "node bin/checkLinks",
+    "clean": "hexo clean",
+    "deploy": "hexo deploy && open 'https://aframe.io/'",
     "dev": "npm run clean && npm run server",
-    "server": "hexo server",
     "generate": "hexo generate",
     "predeploy": "hexo clean",
-    "deploy": "hexo deploy && open 'https://aframe.io/'",
+    "pretest": "multidep multidep.json",
```

Comment:
oops, i was following a guide that was doing this for testing. thanks for catching